WooCommerce/WooCommerceTests/ViewRelated/Products/Media/ProductUIImageLoaderTests.swift

```diff
@@ -46,7 +46,7 @@ final class ProductImagesProviderTests: XCTestCase {
     func testRequestingImageWithPHAsset() {
         let asset = PHAsset()
         let mockPHAssetImageLoader = MockPHAssetImageLoader(imagesByAsset: [asset: testImage])
-        let imagesProvider = DefaultProductImagesProvider(imageService: imageService, phAssetImageLoader: mockPHAssetImageLoader)
+        let imagesProvider = DefaultProductUIImageLoader(imageService: imageService, phAssetImageLoader: mockPHAssetImageLoader)
```

Comment:
We missed this one. 🙂 

```suggestion
        let imageLoader = DefaultProductUIImageLoader(imageService: imageService, phAssetImageLoader: mockPHAssetImageLoader)
```

Reply:
updated in: https://github.com/woocommerce/woocommerce-ios/pull/1914/commits/bef63d1cf1e24112f0cbf95be40b67d224a16658